host_core/lib/host_core/providers/builtin/logging.ex

```diff
@@ -1,7 +1,7 @@
 defmodule HostCore.Providers.Builtin.Logging do
   require Logger
 
-  def invoke(actor, "WriteLog", payload) do
+  def invoke(actor, method, payload) when method == "Logging.WriteLog" or method == "WriteLog" do
```

Comment:
I actually think you can rewrite this to:

`def invoke(actor, method, payload) when method in ["Logging.WriteLog", "WriteLog"] do`